code/modules/cargo/packs.dm

```diff
@@ -1136,6 +1136,18 @@
 	for(var/i in 1 to 49)
 		new /mob/living/simple_animal/butterfly(.)
 
+/datum/supply_pack/organic/critter/crab
+	name = "Crab Rocket"
+	cost = 5000
+	contains = list(/mob/living/simple_animal/crab)
+	crate_name = "look sir free crabs"
+	DropPodOnly = TRUE
+
+/datum/supply_pack/organic/critter/crab/generate()
+	. = ..()
+	for(var/i in 1 to 49)
```

Comment:
Is the gin messing with me or does this have the chance to spawn 49 crabs

Reply:
crab rocket

Reply:
not chance, it always spawns 49 crabs

Reply:
This is fine

Reply:
actually it always spawns 50 crabs because it contains one other crab in the contents